bin/adapt

```diff
@@ -1,3 +1,15 @@
 #!/usr/bin/env node
+var chalk = require("chalk");
+var fs = require('fs');
+var path = require('path');
+var semver = require('semver');
+var curVer = process.versions.node;
+var pkg = JSON.parse(fs.readFileSync(path.join(__dirname, '../package.json')));
+var versRange = pkg.engines.node;
```

Comment:
good catch i intended to use it in the next if block, but it can be removed, I should have linted this :)